.circleci/config.yml

```diff
@@ -2,15 +2,35 @@ version: 2.1
 orbs:
   slack: circleci/slack@3.4.2
 
-jobs:
-  build:
+executors:
+  tap_tester_mongo_4_2:
     docker:
       - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
-      - image: mongo:4.2.0-bionic
+      - image: singerio/mongo:4.2-bionic
         environment:
           MONGO_INITDB_ROOT_USERNAME: dev
           MONGO_INITDB_ROOT_PASSWORD: Password1
-        command: [mongod, --replSet, rs0]
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+  tap_tester_mongo_4_4:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
+      - image: singerio/mongo:4.4-bionic
+        environment:
+          MONGO_INITDB_ROOT_USERNAME: dev
+          MONGO_INITDB_ROOT_PASSWORD: Password1
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+  tap_tester_mongo_5_0:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
+      - image: singerio/mongo:5.0
+        environment:
+          MONGO_INITDB_ROOT_USERNAME: dev
+          MONGO_INITDB_ROOT_PASSWORD: Password1
+        command: [mongod, --replSet, rs0, --keyFile, /opt/mongo/keyfile]
+
+jobs:
+  build:
```

Comment:
Should we rename this to `build_mongo_4_2` so it's clear what each job is running?

Reply:
That's how it was originally but then circle complained that there was no generic "build" job that ran so it waited forever without finishing.  I changed the 4.2 job back to "build" to get around this.  Should we make a fourth empty job for "build"?  Or can we somehow disable the default build check?

Reply:
Circle complained? I forgot the GitHub webhook is probably listening for `build` too so maybe we shouldn't touch this

Reply:
I wonder if they say something about this in their docs

Reply:
Decided to keep it as "build"